integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Tests for the most popular images from docker hub.
+
+source ${BATS_TEST_DIRNAME}/../../metrics/lib/common.bash
+
+setup() {
+	clean_env
+}
+
+@test "create a directory in an aerospike container" {
```

Comment:
This test is extremely useful. However what isn't immediately obvious is *why* each test was chosen (why `mkdir`? Why aerospike?)

Could you rework each `@test` slightly to include a "hint" as to why the test was included? I'd suggest using one or more "tags" in square brackets before the test description. For example, this test might become something like:

```
@test "[volume mkdir] create a directory in an aerospike container"
```

... to suggest that the test is specifically trying to test a `mkdir` on a docker volume.

This hint or tag would be a similar approach to what we do for the hypervisor config script where each configure option specifies a tag (such as `security`, `speed`, `size`). For example:

- https://github.com/kata-containers/packaging/blob/master/scripts/configure-hypervisor.sh#L307

Bonus points if you can find a way to validate the `@test` tags as we do in the hypervisor config script:

- https://github.com/kata-containers/packaging/blob/master/scripts/configure-hypervisor.sh#L21..L29